libraries/mapping/engines/cesium/src/lib/CustomViewer/hooks.ts

```diff
@@ -40,12 +41,15 @@ const useInitializeViewer = (
   const isSecondaryStyle = useShowSecondaryTileset();
   const minZoom = useScreenSpaceCameraControllerMinimumZoomDistance();
   const maxZoom = useScreenSpaceCameraControllerMaximumZoomDistance();
-  const enableCollisionDetection = useScreenSpaceCameraControllerEnableCollisionDetection();
+  const enableCollisionDetection =
+    useScreenSpaceCameraControllerEnableCollisionDetection();
+  const isMode2d = useViewerIsMode2d();
 
   useEffect(() => {
     if (viewer) {
       console.log("HOOK: enable terrain collision detection");
       const scene: Scene = viewer.scene;
+      scene.requestRenderMode = true;
```

Comment:
@d4v3000 should reduce idle draw calls dramatically
https://cesium.com/blog/2018/01/24/cesium-scene-rendering-performance/
https://cesium.com/learn/cesiumjs/ref-doc/Scene.html#requestRenderMode